.github/workflows/pythonbuild.yml

```diff
@@ -78,33 +78,39 @@ jobs:
         with:
           path: ./wheelhouse/*.whl
 
-  build_wheels_manylinux1:
+  build_wheels_manylinux:
     needs: [test_python]
-    name: Build wheel with manylinux1
+    name: Build wheels on ${{matrix.arch}} for ${{matrix.python_tag}}
     runs-on: ubuntu-18.04
     strategy:
       fail-fast: false
-    env:
-      CIBW_BUILD: cp*
-      CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
-      CIBW_MANYLINUX_I686_IMAGE: manylinux1
+      matrix:
+        arch: [auto, aarch64, ppc64le, s390x]
+        python_tag: [ "cp35-*", "cp36-*", "cp37-*", "cp38-*", "cp39-*"]
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
         with:
           submodules: 'true'
 
       - uses: actions/setup-python@v1
-        name: Install Python
+        name: Install Python 3.8
         with:
-          python-version: '3.7'
+          python-version: "3.8"
 
       - name: Install cibuildwheel
         run: |
-          python -m pip install git+https://github.com/joerick/cibuildwheel.git@f6eaa9f
-      - name: Build wheels
+          python -m pip install cibuildwheel~=1.8.0
```

Comment:
I did this back then to get a newer version that supported Python3.9. I think this should probably be updated to the latest version v1.10.0. Also one of the latest releases apparently added support to crosscompile mac os wheels for arm64 and universal2, while I currently only build x86_64.

Reply:
I do not know whether this would make the file a lot cleaner, but apparently it is possible to use cibuildwheel as a action since v1.9 https://github.com/joerick/cibuildwheel/releases/tag/v1.9.0